modules/ROOT/pages/configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,196 @@
+= Configure Horizontal Autoscaling (HPA) for Runtime Fabric Deployments
+
+Runtime Fabric instances support horizontal scaling of Mule app deployments by initiating additional replicas in response to the signals configured.
```

Comment:
Signals configured is misleading - can make the customer believe that there are multiple signals and that they can configure the signal. In our case only signal supported is CPU and we (mulesoft) control it so they cannot configure it

Reply:
Suggestion: "instances support horizontal scaling of Mule app deployments by spinning up additional replicas or spinning down unneeded replicas in response to the CPU usage as a signal."